packages/extension-ui/src/hooks/useMetadata.ts

```diff
@@ -6,13 +6,14 @@ import type { Chain } from '@polkadot/extension-chains/types';
 import { useEffect, useState } from 'react';
 
 import { getMetadata } from '../messaging';
+import BN from "bn.js";
```

Comment:
nit (and other places)
```suggestion
import BN from 'bn.js';
```